### ISSUE TITLE
How to reload only one specific Test Suite?

### ISSUE THREAD

**Issue by daneshmandi** [hbenl/vscode-test-explorer#82]
Is there a way to only reload one specific test suite rather than reload all tests?

If not, can you consider as a feature request. Like a right click context menu on a single test suite(file) for instance.

**Comment by connorshea** [comment]
I don't believe this is possible with the current way adapters are built. Loads/reloads are sent to the adapters and they're only meant to request that _all_ tests be reloaded, there's not a way to do partial reloads currently.

**Comment by matepek** [comment]
@daneshmandi Why is this important? What is your problem actually?

**Comment by jaywick** [comment]
@matepek I'm looking for this feature as well, the main reason is because the project I'm on has thousands of tests in a monorepo. We run our tests (in our case using jest) on folders or specific files.
